app/controllers/billboards_controller.rb

```diff
@@ -0,0 +1,91 @@
+class BillboardsController < ApplicationController
+	
+  @@show_free = false
```

Comment:
Don't use class variables. Most probably you should put this value into the session